Makefile.include

```diff
@@ -508,7 +508,7 @@ $(RIOTBUILD_CONFIG_HEADER_C):
 	@mkdir -p '$(dir $@)'
 	$(AD)'$(RIOTBASE)/dist/tools/genconfigheader/genconfigheader.sh' '$@' $(CFLAGS_WITH_MACROS)
 
-CFLAGS_WITH_MACROS := $(CFLAGS)
+CFLAGS_WITH_MACROS := $(CFLAGS) $(LTOFLAGS)
```

Comment:
This may cause undesired side effects by including the LTOFLAGS defines in the riotbuild.h header.
e.g. `LTOFLAGS="-flto -DUSING_LTO=1` will define USING_LTO=1 for all files, not just the LTO files.